indico/migrations/versions/20220225_1335_ef7a8b2e6737_add_visibility_options_to_menu_entries.py

```diff
@@ -0,0 +1,44 @@
+"""Add visibility options to menu entries
+
+Revision ID: ef7a8b2e6737
+Revises: 3dafee32ba7d
+Create Date: 2022-02-25 13:35:15.089546
+"""
+
+from enum import Enum
+
+import sqlalchemy as sa
+from alembic import op
+
+from indico.core.db.sqlalchemy import PyIntEnum
+
+
+# revision identifiers, used by Alembic.
+revision = 'ef7a8b2e6737'
+down_revision = '82fb6c6ac6db'
+branch_labels = None
+depends_on = None
+
+
+class _MenuEntryVisibility(int, Enum):
+    everyone = 1
+    registered_users = 2
+    speakers = 3
+
+
+def upgrade():
+    op.add_column('menu_entries',
+                  sa.Column('visibility', PyIntEnum(_MenuEntryVisibility), server_default='1', nullable=False),
```

Comment:
hm... `visibility` or `access`...? i don't have a strong opinion for either option though